README.md

```diff
@@ -0,0 +1,163 @@
+# selenium-sandbox
+
+easily mock a web application accessed through selenium. Contains also an environment for integrating with jest.
+
+## Getting started
+
+### 0. Install
+
+    yarn add -D @4c/selenium-sandbox
+
+### 1. Add the sandbox to your test client
+
+First, we need to instrument our application to use the sandbox environment. Create a module (let's call it `injectSandbox.js`)
+
+```js
+import sandbox from '@4c/selenium-sandbox/lib/sandbox';
```

Comment:
Are the imports split like this for browser and server? If so I think it'll be "cleaner" to use the browser main field for browser code and then everything can be a main import, e.g. `import sandbox from 'selenium-sandbox'`

Reply:
not sure what is more "main" than the other, because they are equally necessary parts 🤔

Reply:
It can't be isomorphic, one thing needs to run in a selenium context, the other on the web. Not sure I follow you. Also, they are both "clients" in a way so I think that's confusing. What's the server? If there is a server, that's webdriver. None of these two

Reply:
right, so i mean, i'm disagreeing with @jquense here in that i don't think we should use the `browser` entry point. best to be explicit. `/webdriver` and `/browser` might be good

Reply:
Fair enough, we can have two imporrs, I like webdriver and browser

Reply:
but that's the current case then. one is called `sandbox` and the other `selenium`... should I just rename them to `browser` and `webdriver`?

Reply:
I would like to just remove the `/lib/` segment. naming of each file is less important to me

Reply:
isn't that a pain with the current setup, or am I missing something trivial?

Reply:
I could just compile to `.` instead of `lib` i guess 🤔